bin/setenv-docker-customize.sh

```diff
@@ -651,10 +651,16 @@ else
     echo "# no add-on to install from EXO_ADDONS_LIST environment variable."
   else
     echo "# installing add-ons from EXO_ADDONS_LIST environment variable:"
+    _ADDON_COUNTER=0
     echo ${EXO_ADDONS_LIST} | tr ',' '\n' | while read _addon ; do
       if [ -n "${_addon}" ]; then
+        _ADDON_COUNTER=$((_ADDON_COUNTER+1))
         # Install addon
-        ${EXO_APP_DIR}/addon install ${_ADDON_MGR_OPTIONS:-} ${_ADDON_MGR_OPTION_CATALOG:-} ${_addon} --force --batch-mode
+        if [ ${_ADDON_COUNTER} -eq "1" ]; then 
```

Comment:
Is it needed in case of container stop/start? For the stop/up a new container is deployed. So, where the information is cached in this case?

Reply:
ah ok, in case of restart due to failure. is it?

Reply:
> ah ok, in case of restart due to failure. is it?

Yep, in case of container restart on error